src/composable/ConditionalOrder.ts

```diff
@@ -300,7 +300,7 @@ export abstract class ConditionalOrder<D, S> {
    * @param params owner context, to be able to check if the order is authorised
    * @returns true if the owner authorised the order, false otherwise.
    */
-  public isAuthorized(params: OwnerContext): Promise<boolean> {
+  public isAuthorised(params: OwnerContext): Promise<boolean> {
```

Comment:
irrelevant change, but i tried to be coherent with the other spellings of the word

Reply:
No problems, I generally used UK English in documentation, and resisted making changes to functions to avoid potentially affecting downstream users until another major semver bump. But given we're still at `rc`, no issues :sweat_smile:

Reply:
https://github.com/cowprotocol/cow-sdk/pull/160